CONTRIBUTING.md

```diff
@@ -74,7 +76,7 @@ git commit # type a good commit message
 
 6. Bring master up-to-date and rebase
 
-If you are preparing your first pull request on this branch, rebase to master
+Since your local fork is likely out of sync with the Decred, you'll need to realign everything and clean up your commits.

```

Comment:
I'm pretty sure I know what you are saying here, but I find the phrasing with "fork" and "realign" a little imprecise.  When you are saying out of sync, I guess you mean your local `master` branch being out of date w.r.t. the remote `master` branch of Decred's fork/repo.  If your local `master` is actually tracking a `master` branch in your actual *fork* depends on your workflow.  Personally I have local `master` track the decred remote branch, not the remote `master` in my fork.  In fact, I do not even use the `master` branch in my fork.  I might have even deleted it.